sdk/servicebus/Microsoft.Azure.ServiceBus/src/Microsoft.Azure.ServiceBus.csproj

```diff
@@ -32,9 +32,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" />
     <PackageReference Include="Microsoft.Azure.Amqp" />
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" />
+    <PackageReference Include="Newtonsoft.Json" VersionOverride="13.0.1"/>
```

Comment:
Why are we using a VersionOverride here? This is the same version listed in the packages.data.props.

Reply:
Nevermind, I see this is the Track 1 library.

Reply:
I'd still ask the same question.  Why override locally rather than at the root level?

Reply:
I did this for the same reason as above. The aim is not to upgrade dependency of every package. Note, w/o the assembly updates runtime resolution was still picking up 13.0.1 and w/o including this we were downgrading our dependencies to 10 version.

My thought here was lets only upgrade the minimum necessary for 2 reasons
1. Restrict the area of change.
2. Given this is T1 and will be obsolete soon, I don't see a value in doing it across the board for t1 ecosystem.

Reply:
But this dependency with a vulnerable version already exists in the packages file:  
https://github.com/Azure/azure-sdk-for-net/blob/main/eng/Packages.Data.props#L31

I don't understand the advantage to doing this locally.

Reply:
Why not just bump all the T1 packages to a safe version?  Why let a vulnerable version still exist as the baseline?

Reply:
It is not the newtonsoft that is vulnerable. If it were I would absolutely bump that too.